h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -101,26 +130,30 @@ export default function AnnotationCard({ annotation }: AnnotationCardProps) {
             </ul>
           )}
 
-          <footer className="flex items-end justify-between">
-            <ModerationStatusSelect
-              onChange={/* istanbul ignore next */ () => {}}
-              selected={annotation.moderation_status}
-              mode="select"
-              alignListbox="left"
-            />
-            <div className="flex items-center gap-1">
-              <Link
-                variant="text-light"
-                href={annotation.links.incontext}
-                target="_blank"
-                title="See in context"
-                aria-label="See in context"
-                data-testid="context-link"
-              >
-                <ExternalIcon />
-              </Link>
-              <AnnotationShareControl annotation={annotation} group={group} />
+          <footer className="flex flex-col gap-2">
+            <div className="flex items-end justify-between">
+              <ModerationStatusSelect
+                onChange={onModerationStatusChange}
+                disabled={selectDisabled}
+                selected={annotation.moderation_status}
+                mode="select"
+                alignListbox="left"
+              />
+              <div className="flex items-center gap-1">
+                <Link
+                  variant="text-light"
+                  href={annotation.links.incontext}
+                  target="_blank"
+                  title="See in context"
+                  aria-label="See in context"
+                  data-testid="context-link"
+                >
+                  <ExternalIcon />
+                </Link>
+                <AnnotationShareControl annotation={annotation} group={group} />
+              </div>
             </div>
+            {errorUpdating && <div>{errorUpdating.message}</div>}
```

Comment:
I assume this wrapper `<div>` is temporary, but it isn't strictly required since items in a flex container get implicitly wrapped in a `div`-like container.

Reply:
It was sort-of temporary, but it has more attributes now (classes, testid, etc)